frontend/src/RecordingPageComp/RecordingComp.jsx

```diff
@@ -3,6 +3,7 @@
 import { AudioRecorder, useAudioRecorder } from "react-audio-voice-recorder";
 import { useContext, useEffect, useRef } from "react";
 import { UserContext } from "../UserContext.js";
+import "./Microphone.css";
```

Comment:
I would consider naming the CSS file the same as the component file, so when looking at the repo, it's clear what stylesheet belongs to which component